uint8 Status             # Byte0
uint8 RegisterIndex      # Byte1
uint8 RegisterFormat     # Byte2
uint32 RegisterValue     # packed Value0..3 (32-bit) Byte3...6 
uint8 MessageIdentifier  # Byte7
